packages/app/src/runner/index.ts

```diff
@@ -305,12 +304,12 @@ function runSpecE2E (spec: SpecFile) {
  * This only needs to happen once, prior to running the first spec.
  */
 async function initialize () {
+  await dfd.promise
```

Comment:
The trick here is by using a deferred promise, we will only wait for this IF the bundle has not been injected. Otherwise, full steam ahead, no waiting or blocking the UI.